dataland-api-key-manager/src/main/kotlin/org/dataland/datalandapikeymanager/controller/ApiKeyController.kt

```diff
@@ -22,6 +22,10 @@ class ApiKeyController(
         return ResponseEntity.ok(apiKeyManager.generateNewApiKey(daysValid))
     }
 
+    override fun getApiKeyMetaInfoForUser(keycloakUserId: String): ResponseEntity<ApiKeyMetaInfo> {
+        return ResponseEntity.ok(apiKeyManager.getApiKeyMetaInfoForFrontendUser(keycloakUserId))
```

Comment:
With this implementation, any user with role "ROLE_USER" gout get API-Key-Meta-Infos about any other user.

Reply:
This should not be the case anymore, since you can't pass a keycloakUserId to this method anymore.